src/xenia/ui/vulkan/vulkan_instance.cc

```diff
@@ -227,10 +264,21 @@ bool VulkanInstance::CreateInstance() {
 
   XELOGVK("Initializing application instance...");
 
+  VkDebugReportCallbackCreateInfoEXT debug_info;
+  debug_info.sType = VK_STRUCTURE_TYPE_DEBUG_REPORT_CREATE_INFO_EXT;
+  debug_info.pNext = nullptr;
+  // TODO(benvanik): flags to set these.
+  debug_info.flags =
+      VK_DEBUG_REPORT_INFORMATION_BIT_EXT | VK_DEBUG_REPORT_WARNING_BIT_EXT |
+      VK_DEBUG_REPORT_PERFORMANCE_WARNING_BIT_EXT |
+      VK_DEBUG_REPORT_ERROR_BIT_EXT | VK_DEBUG_REPORT_DEBUG_BIT_EXT;
+  debug_info.pfnCallback = &DebugMessageCallback;
+  debug_info.pUserData = this;
+
   // TODO(benvanik): use GetEntryInfo?
   VkApplicationInfo application_info;
   application_info.sType = VK_STRUCTURE_TYPE_APPLICATION_INFO;
-  application_info.pNext = nullptr;
+  application_info.pNext = &debug_info;
```

Comment:
Why change initialization of debug report stuff to this from `EnableDebugValidation();`?

Reply:
I am going to have to revisit this commit.
At the time, it seemed that `EnableDebugValidation()` was not logging any validation layers and this commit fixed that.

Reply:
Moved to #1431